site/app/templates/grading/Images.twig

```diff
@@ -1,6 +1,12 @@
 <div class="content">
-    {% if hasInstructorPermission %}
+    {% if has_full_access %}
         <div style="float: right; margin-bottom: 20px;">
+            {% set url = core.buildCourseUrl(['student_photos']) %}
```

Comment:
I think we should not use core in twig files and for this one, we can just use template values

Reply:
Removed core.